cmd/antrea-controller/controller.go

```diff
@@ -155,6 +157,8 @@ func run(o *Options) error {
 		networkPolicyStore,
 		groupStore)
 
+	externalNodeController := externalnode.NewExternalNodeController(crdClient, externalNodeInformer, eeInformer)
```

Comment:
Could we put the code under the external node feature gate?

Reply:
OK, I will update this part and add a feature gate on controller side.